### PR TITLE
feat(port-healthcheck): update on port settings for health checks

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-740

2023-08-02-180152_1271x861_scrot
2023-08-02-180201_1417x864_scrot

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Write comments that add new information that doesn’t exist in the code

File: libs/shared/console-shared/src/lib/flow-create-port/utils/port-healthcheck.spec.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { PortProtocolEnum, type ProbeType } from 'qovery-typescript-axios'
+import { isMatchingHealthCheck } from './port-healthcheck'
+
+describe('Port HealthCheck utils', () => {
+  describe('isMatchingHealthCheck', () => {
```
Comment (RemiBonnet) [inline libs/shared/console-shared/src/lib/flow-create-port/utils/port-healthcheck.spec.ts:5]: Why you have double describe, is not better to have only one in this case?

File: libs/shared/ui/src/lib/components/banner-box/banner-box.tsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ export function BannerBox(props: BannerBoxProps) {
         />
       </div>
 
-      <div>
+      <div className="flex flex-col justify-center">
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/banner-box/banner-box.tsx:63]: Really necessary ? If needed I think you can directly put `flex items-center`

File: libs/shared/ui/src/lib/components/inputs/input-select/input-select.tsx [hunk 3]
```diff
@@ -106,7 +106,7 @@ export function InputSelect({
 
   const MenuList = (props: MenuListProps<Value, true, GroupBase<Value>>) => (
     <components.MenuList {...props}>
-      {props.children}
+      <div role="listbox">{props.children}</div>
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/inputs/input-select/input-select.tsx:109]: 👍

File: libs/shared/ui/src/lib/components/inputs/input-select/input-select.tsx [hunk 4]
```diff
@@ -124,24 +124,29 @@ export function InputSelect({
...
-      ) : (
-        <Icon name={IconAwesomeEnum.CHECK} className="opacity-0" />
-      )}
-      <Tooltip content={props.label}>
-        <label className="ml-2 truncate">{props.label}</label>
-      </Tooltip>
-    </components.Option>
-  )
+  const Option = (props: OptionProps<Value, true, GroupBase<Value>>) => {
+    const id = useId()
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/inputs/input-select/input-select.tsx:128]: 👍

**Review by RemiBonnet — APPROVED** [review]
Thanks for this PR @ctjhoa 💯, small optional comments

File: libs/shared/console-shared/src/lib/flow-create-port/utils/port-healthcheck.spec.ts (hunk 1 above)
Comment (ctjhoa) [reply libs/shared/console-shared/src/lib/flow-create-port/utils/port-healthcheck.spec.ts]: https://github.com/Qovery/console/pull/728/commits/b4b33c1a8de70d4ba343d9f67012cd2933417a03

**Pull request merged**